### PR TITLE
Fix getting template source and update deps

### LINKED ISSUE #967: 500 error after update
when I log on admin area I receive this error

Argument 1 passed to App\Twig\SourceCodeExtension::getTemplateSource() must be an instance of Twig\TemplateWrapper, instance of __TwigTemplate_2cbf7fc5651a791395a7872583630b929ff0fd2ae3fa9cc8e5edc91b3a4ac77e given, called in /var/www/ ....

could you recheck in your side if this admin area it is working

### PR DESCRIPTION
**Pull request by bocharsky-bw**
Fix #967 and some vulnerability issues in deps

### CODE REVIEW

File: src/Twig/SourceCodeExtension.php [hunk 1]
```diff
@@ -92,8 +93,17 @@ private function getCallableReflector(callable $callable): \ReflectionFunctionAb
         return new \ReflectionFunction($callable);
     }
 
-    private function getTemplateSource(TemplateWrapper $template): array
+    private function getTemplateSource($template): array
     {
+        if (!$template instanceof TemplateWrapper && !$template instanceof Template) {
+            throw new \Exception(sprintf(
```
Comment (yceruto) [inline src/Twig/SourceCodeExtension.php]: I'm not sure this exception is needed... this function is private/local and `$twig->resolveTemplate($template)` ony returns `TemplateWrapper` or `Template`.

I'd suggest adding a PHPDoc block instead.

**Comment by yceruto** [comment]
Shouldn't we also update this line?
https://github.com/symfony/demo/blob/4bc69f63c950a31a046b82d7fef19de4c831c4af/composer.json#L33

File: src/Twig/SourceCodeExtension.php (hunk 1 above)
Comment (bocharsky-bw) [reply src/Twig/SourceCodeExtension.php]: I'm fine with PHPDoc, just was coding defensively... Let's wait for more feedback then

**Comment by bocharsky-bw** [comment]
> Shouldn't we also update this line?

Hm, don't think we needed. Updated to what?

File: src/Twig/SourceCodeExtension.php (hunk 1 above)
Comment (javiereguiluz) [reply src/Twig/SourceCodeExtension.php]: I agree with Yonel on this one.

Comment (bocharsky-bw) [reply src/Twig/SourceCodeExtension.php]: OK, it's simpler now 👍

**Comment by javiereguiluz** [comment]
Thank you Victor; it's merged now.

**Pull request closed without merging**